store/sqlstore/supplier_test.go

```diff
@@ -174,8 +174,9 @@ func TestRecycleDBConns(t *testing.T) {
 				wg.Add(1)
 				go func(table string) {
 					defer wg.Done()
-					res := supplier.DoesTableExist(table)
-					assert.True(t, res)
+					query := `SELECT count(*) FROM ` + table
+					_, err := supplier.GetMaster().SelectInt(query)
+					assert.Nil(t, err)
```

Comment:
Since `SelectInt` returns an `error`, we could use `assert.NoError(t, err)` here instead of the purely-nil check required for `*model.AppError`.

Reply:
Good one. We should probably have a vet check to catch other cases like this throughout the codebase.